boards/btt-skr-mini-e3-30/board-definition.json

```diff
@@ -7,6 +7,7 @@
 	"compileScript": "compile.sh",
 	"flashScript": "flash.sh",
 	"driverCount": 4,
+	"driverVoltages": [24, 36],
```

Comment:
None of the electronics on this board supports 36V and there's no separate motor supply